src/client.rs

```diff
@@ -820,27 +851,27 @@ mod tests {
     #[meilisearch_test]
     async fn test_get_keys(client: Client) {
         let keys = client.get_keys().await.unwrap();
-        assert!(keys.len() >= 2);
-        assert!(keys.iter().any(
-            |k| k.description != "Default Search API Key (Use it to search from the frontend)"
-        ));
-        assert!(keys.iter().any(
-            |k| k.description != "Default Admin API Key (Use it for all other operations. Caution! Do not use it on a public frontend)"
+        assert!(keys.results.len() >= 2);
+        assert!(keys.results.iter().any(|k| k.description
+            != Some("Default Search API Key (Use it to search from the frontend)".to_string())));
+        assert!(keys.results.iter().any(
+            |k| k.description != Some("Default Admin API Key (Use it for all other operations. Caution! Do not use it on a public frontend)".to_string())
```

Comment:
Once again, since the keys are now paginated, there is some chance we’ll never find these two keys :cry: 
Sadly, I don’t think we can test anything more than the len.